### ISSUE TITLE
Upgrade to Manifest V3

### ISSUE THREAD

**Issue by YunzheZJU** [YunzheZJU/holo-schedule#110]
Chrome Web Store requires all new submitted extensions to use the latest manifest version.

This task should be finished before Jan 17th, 2022, or new versions would fail to pass the check from the store.

**Comment by YunzheZJU** [comment]
Merged into master da1e916.
